### PR TITLE
Sample - Updating WLS binaries and using old domain

### PR DESCRIPTION
**Pull request by ddsharpe**
A sample that describes the use case of updating the WebLogic or Java binaries without disturbing the domain configuration.

### CODE REVIEW

File: samples/copy-domain/Readme.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Example Updating the Oracle Binaries for My Domain
```
Comment (markxnelson) [inline samples/copy-domain/Readme.md]: "Example: " or "Example of", and fix the capitals - I think Oracle Doc standard is lower case after first word, except for trademarks etc - check with @rosemarymarano

File: samples/copy-domain/Readme.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+Example Updating the Oracle Binaries for My Domain 
+==================================================
+Docker images are built in immutable layers, but I needed to patch the Oracle Home for my domain.
```
Comment (markxnelson) [inline samples/copy-domain/Readme.md]: You should not use first person

File: samples/copy-domain/Readme.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+In a typical environment, I would run OPatch against my Oracle Home and be done.  But with a Docker
```
Comment (markxnelson) [inline samples/copy-domain/Readme.md]: "and be done" is colloquial

File: samples/copy-domain/Readme.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+image, I cannot do that because the layer with the original Oracle Home is immutable.  I could add 
+another layer with the patch, but I cannot change the original Oracle Home layer.  Adding a patch in 
+a new layer means that the resulting Docker image contains all the original files plus all the files that
+the patch replaced.  And, I need to do this same update for all my domain images.  In looking for a
+better solution, I found that I could copy my old domain from one Docker image to another.  All I had
+to do was keep my base image and domain image separate.
+
+Here is an example of what I mean. 
+### Setup
+Load the WebLogic Image Tool cache with pointers to the installers:
```
Comment (markxnelson) [inline samples/copy-domain/Readme.md]: Load the installers into the WebLogic Image Tool cache using these commands:

File: samples/copy-domain/Readme.md (hunk 1 above)
Comment (rosemarymarano) [reply samples/copy-domain/Readme.md]: Agree.

File: samples/copy-domain/Readme.md (hunk 2 above)
Comment (rosemarymarano) [reply samples/copy-domain/Readme.md]: Also, true.

**Comment by ddsharpe** [comment]
moving sample to a product feature

**Pull request closed without merging**